Adapters/Vungle/ISVungleAdapter/ISVungleBannerAdapterRouter.m

```diff
@@ -0,0 +1,212 @@
+//
+//  ISVungleBannerAdapterRouter.m
+//  ISVungleAdapter
+//
+//  Copyright © 2020 ironSource. All rights reserved.
+//
+
+#import "ISVungleBannerAdapterRouter.h"
+#import "ISVungleAdapter.h"
+
+@interface ISVungleBannerAdapterRouter ()
+
+@property (nonatomic, strong, nullable) NSString *bidPayload;
+
+@end
+
+@implementation ISVungleBannerAdapterRouter
+
+- (instancetype)initWithPlacementID:(NSString *)placementID
+                      parentAdapter:(ISVungleAdapter *)parentAdapter
+                           delegate:(id<ISBannerAdapterDelegate>)delegate {
+    if (self = [super init]) {
+        _placementID = placementID;
+        _parentAdapter = parentAdapter;
+        _delegate = delegate;
+        _bannerState = UNKNOWN;
+        _bidPayload = nil;
+        self.bannerSize = BannerSizeRegular;
```

Comment:
This should follow the pattern `_bannerSize`

Reply:
@stanley-vungle Thanks for pointing out this issue, updated. :)